sentry_sdk/utils.py

```diff
@@ -191,8 +190,14 @@ def capture_internal_exceptions():
 
 def capture_internal_exception(exc_info):
     # type: (ExcInfo) -> None
+    """
+    Capture an exception that is likely caused by a bug in the SDK
+    itself.
+
+    These exceptions do not end up in Sentry and are just logged instead.
+    """
     if sentry_sdk.get_client().is_active():
-        sentry_sdk.Scope._capture_internal_exception(exc_info)
```

Comment:
Maybe we should get rid of `Scope._capture_internal_exception()` as a whole and use `utils.capture_internal_exception()` everywhere?